app/javascript/dashboard/components/widgets/Avatar.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div
+    class="avatar-container"
+    :style="[style, customStyle]"
+    aria-hidden="true"
+  >
+    <span>{{ userInitial }}</span>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Avatar',
+  props: {
+    username: {
+      type: String,
+    },
+    backgroundColor: {
+      type: String,
+    },
+    color: {
+      type: String,
+    },
+    customStyle: {
+      type: Object,
+    },
+    size: {
+      type: Number,
+      default: 40,
+    },
+    src: {
+      type: String,
+    },
+    rounded: {
+      type: Boolean,
+      default: true,
+    },
+  },
+  computed: {
+    style() {
+      return {
+        display: 'flex',
+        width: `${this.size}px`,
+        height: `${this.size}px`,
+        borderRadius: this.rounded ? '50%' : 0,
+        lineHeight: `${this.size + Math.floor(this.size / 20)}px`,
+        fontWeight: 'bold',
```

Comment:
Can we move the constant attributes to class
```
<style lang="scss">
.avatar--container {
  display:flex // etc
}
</style>
```

Reply:
Done!